db/indexes.go

```diff
@@ -118,17 +118,17 @@ var (
 	// Queries used to check readiness on startup.  Only required for critical indexes.
 	readinessQueries = map[SGIndexType]string{
 		IndexAccess: "SELECT $sync.access.foo as val " +
-			"FROM `%s` " +
+			"FROM %s AS %s " +
```

Comment:
@adamcfraser Question for you:

Technically we're changing the indicies when running on the `_default` collection.
Correct me if I'm wrong, but these changes should be safe not to bump the index version for a rebuild since all we're doing is specifying a keyspace alias, which is required when `$_keyspace` is a fully qualified collection.

I don't think there are any upgrade/mixed SG version concerns either (3.1 index/3.0 query and vice versa) but I do need to manually verify this.

Reply:
You would need to validate that the updated queries (that include `from %s as %s`) work against the 3.0 index definition.  If that works, I agree there shouldn't be any mixed concerns.

Reply:
It works as expected in mixed versions (these aliased queries can run on a 3.0 index, and the other way around if we needed to support downgrade)